src/forms.css

```diff
@@ -1,3 +1,11 @@
+/**
+ * Indicates that a label will shift focus to its associated control.
+ */
+
+label[for] {
+  cursor: pointer;
+}
```

Comment:
what's the deal for `disabled` fields, where it's weird that the label would appear clickable but the focus wouldn't actually shift I don't think, as `disabled` elements aren't focusable